pkg/plugin/aggregation/run.go

```diff
@@ -80,7 +81,11 @@ func Run(client kubernetes.Interface, plugins []plugin.Interface, cfg plugin.Agg
 	// TODO: there are other places that iterate through the CoreV1.Nodes API
 	// call, we should only do this in one place and cache it.
 	nodes, err := client.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
-	if err != nil {
+	switch {
+	case apierrors.IsUnauthorized(err) || apierrors.IsForbidden(err):
+		logrus.Warningf("Unauthorized to query cluster nodes; continuing with empty node list. Daemonset plugins will be ignored as a result.")
```

Comment:
This is accurate; they will be ignored. Not failed, which is an important distinction. Not sure if there is a way to mark the plugin as failed since it is a daemonset and we dont have the node names.